packages/manager/modules/sign-up/src/service.js

```diff
@@ -18,6 +20,20 @@ export default class SignUpService {
       .then(({ data }) => data);
   }
 
+  getCreationRulesParams() {
+    return this.$http
```

Comment:
No use of `ovh-api-services` in the module ?

Reply:
Nope, as I have only few calls to do, no need to import full `ovh-api-services`.